prow/config/plugins.yaml

```diff
@@ -170,24 +170,24 @@ triggers:
 
 milestone_applier:
   metal3-io/cluster-api-provider-metal3:
-    main: v1.9
-    release-1.8: v1.8
-    release-1.7: v1.7
-    release-1.6: v1.6
+    main: "CAPM3 - v1.9"
+    release-1.8: "CAPM3 - v1.8"
+    release-1.7: "CAPM3 - v1.7"
+    release-1.6: "CAPM3 - v1.6"
   metal3-io/baremetal-operator:
-    main: v0.9
-    release-0.8: v0.8
-    release-0.7: v0.7
-    release-0.6: v0.6
-    release-0.5: v0.5
+    main: "BMO - v0.9"
+    release-0.8: "BMO - v0.8"
+    release-0.7: "BMO - v0.7"
```

Comment:
There is no BMO 0.7

Reply:
No there was an issue during release there is a comment on the milestone-

Reply:
Hmm but we have a branch and milestone. Should we remove it from here because we don't expect to use it or do we just keep it?

Reply:
We should not have milestone, and branch is locked, so it should not be mentioned anywhere.

Reply:
Makes sense. I will remove it from here.

Reply:
We have only branch no milestone but yes IMO we can get rid of that 0,7 branch.